### DISCUSSION TITLE
Support for "sites" that are moz-extension rather than https

### DISCUSSION THREAD

**Discussion by Tosh0kan** [filips123/PWAsForFirefox#513] [Ideas]
I have an extension for (Yomitan) for Firefox that is basically a Japanese dictionary. It's extremely useful for when I'm doing translation work, and it would be amazing to be able to use it without the tabs and address bar and such. Thing is, it runs as a `moz-extension://` rather tha `https`, so PWAsForFirefox can't create a web app for it. Would it be possible to add this functionality?

**Comment by filips123** [comment]
Unfortunately, I think this isn't possible to implement currently. Extensions can't access other extensions' pages, so PWAsForFirefox can't obtain page properties to generate a manifest. And even if it could, the native program couldn't access things like icons, as they are bundles inside the extension.

Maybe I can try to add an option to install arbitrary URL as a PWA in the future, but I'm not sure if that would work with extensions.

**Comment by Tosh0kan** [comment]
I see. It would be amazing if possible, but thank for your explanation nonetheless.
